
  0%|                                                                                                                                                                                                                                                     | 0/23795 [00:00<?, ?it/s]Traceback (most recent call last):
  File "/home/vince/embedding_training/./run_mlm.py", line 826, in <module>
    main()
  File "/home/vince/embedding_training/./run_mlm.py", line 764, in main
    train_result = trainer.train(resume_from_checkpoint=checkpoint)
  File "/home/vince/miniconda3/envs/distil/lib/python3.10/site-packages/transformers/trainer.py", line 1530, in train
    return inner_training_loop(
  File "/home/vince/miniconda3/envs/distil/lib/python3.10/site-packages/transformers/trainer.py", line 1836, in _inner_training_loop
    for step, inputs in enumerate(epoch_iterator):
  File "/home/vince/miniconda3/envs/distil/lib/python3.10/site-packages/accelerate/data_loader.py", line 451, in __iter__
    current_batch = next(dataloader_iter)
  File "/home/vince/miniconda3/envs/distil/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 631, in __next__
    data = self._next_data()
  File "/home/vince/miniconda3/envs/distil/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 1329, in _next_data
    idx, data = self._get_data()
  File "/home/vince/miniconda3/envs/distil/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 1285, in _get_data
    success, data = self._try_get_data()
  File "/home/vince/miniconda3/envs/distil/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 1133, in _try_get_data
    data = self._data_queue.get(timeout=timeout)
  File "/home/vince/miniconda3/envs/distil/lib/python3.10/queue.py", line 180, in get
    self.not_empty.wait(remaining)
  File "/home/vince/miniconda3/envs/distil/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
